app/bundles/InstallBundle/Configurator/Form/EmailStepType.php

```diff
@@ -168,16 +174,11 @@ public function buildForm(FormBuilderInterface $builder, array $options)
             'mailer_auth_mode',
             'choice',
             [
-                'choice_list' => new ChoiceList(
-                    [
-                        'plain',
-                        'login',
-                        'cram-md5',
-                    ],
+                'choice_list' => new ArrayChoiceList(
```

Comment:
Can we simply use 
```suggestion
                'choices' => [
```
instead of the choice_list property? I don't see such option in the docs for Symfony 3.4: https://symfony.com/doc/3.4/reference/forms/types/choice.html

And please, switch values and labels.